packages/@angular/cli/commands/help.ts

```diff
@@ -56,7 +56,7 @@ const HelpCommand = Command.extend({
 
       if (rawArgs.length > 0) {
         if (cmd === rawArgs[0]) {
-          this.ui.writeLine(command.printDetailedHelp(commandOptions));
+          this.ui.writeLine(command.printBasicHelp(commandOptions));
```

Comment:
So we changed this line multiple times. Could you investigate where we have detailed help and where we don't, so that we can print detailed if it exists, and basic if it doesn't?